alerts/bmw_connected_drive.markdown

```diff
@@ -1,13 +1,16 @@
 ---
-title: "BMW Connected Drive API shut down"
+title: "BMW Connected Drive API (china) unavailable, other regions restored"
 created: 2021-07-15 14:40:00
 integrations:
   - bmw_connected_drive
 github_issue: https://github.com/home-assistant/core/issues/53977
-homeassistant: ">0.64.0"
+homeassistant: ">0.64.0, <2021.8.5"
```

Comment:
I think this is incorrect, as it's not fixed fully?

It's kinda confusing with that specific range, as it indicates it's fixed, but it is not? If it is fixed, I would say remove it, or else don't state its fixed in that version.

```suggestion
homeassistant: ">0.64.0"
```

Reply:
It is fixed with 2021.8.5 for everyone except mainland China. 
But then I'd rather go with your suggestion as information about the fix is available in the text (and header).